app/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id(BuildPlugins.androidApplication)
+    id(BuildPlugins.kotlinAndroid)
+    id(BuildPlugins.kotlinAndroidExtensions)
+}
+
+android {
+    compileSdkVersion(AndroidSdk.compile)
+
+    defaultConfig {
+        applicationId = appId
+        minSdkVersion(AndroidSdk.min)
+        targetSdkVersion(AndroidSdk.target)
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = TestLibraries.testRunner
+    }
+
+    buildTypes {
```

Comment:
Moving further into this project, it'd be good to make this behaviour scalable incase we want to start adding new modules to the application. You can achieve this by applying this particular behaviour as a plugin.